docs/how-to/deploy/deploy-anywhere.md

```diff
@@ -69,15 +69,49 @@ Make sure that the model is of a correct type (not `k8s`):
 juju show-model | yq '.[].type'
 ```
 
-## Deploy Charmed Apache Kafka
+## Deploy Charmed Apache Kafka for production
```

Comment:
Suggestion: I don't think we have to be that specific here and below.

Reply:
I would like to make clear the two modes we support. Both are important to share.
In the Tutorial however, for convenience we only include one mode (though I did give a bit more detail there).

I want for a user to think "How do I deploy the thing?", look at the 'How To' section, see "Deploy anywhere" and have all the information they need for both use-cases.

If you think it's too much, do you have a thought on how to achieve the above in a different way?

Reply:
OK, let's give it a try. I think I'm growing into this.